preparation.py

```diff
@@ -27,13 +27,17 @@ def overlay_masks(data_dir, dataset, target_dir, is_small=False):
     for image_id in tqdm(image_ids):
         image = coco.loadImgs(image_id)[0]
         image_size = [image["height"], image["width"]]
```

Comment:
@apyskir it is sugested to use immutable in cases like this so I would definitely go with (h,w). Think about it. What are the benefits of using one over the other.

Reply:
Ok, I fixed it.